drivers/soundwire/intel.c

```diff
@@ -1042,6 +1042,7 @@ static int sdw_master_read_intel_prop(struct sdw_bus *bus)
 				 "intel-sdw-ip-clock",
 				 &prop->mclk_freq);
 
+	prop->mclk_freq /= 2;
```

Comment:
Is this ok for upstream or should we modify the original value in property? @RanderWang @plbossart

Reply:
well the values are in the BIOS so it's impractical/impossible to change them. So we have to fix this here.
Note that I will only send this upstream when we have a better explanation for the clock settings, there's a bit of black magic for now.